samples/BasicXrApp/BasicXrApp_win32.vcxproj

```diff
@@ -70,7 +68,7 @@
     <Link>
       <SubSystem>Windows</SubSystem>
       <GenerateWindowsMetadata>false</GenerateWindowsMetadata>
-      <AdditionalDependencies>shlwapi.lib;advapi32.lib;windowsapp.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>d3d11.lib;dxgi.lib;kernel32.lib;user32.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
You don't need these libs in win32 project, but you do need windowsapp.lib; for winrt com_ptr